packages/lib/server/repository/eventType.ts

```diff
@@ -99,12 +99,6 @@ export class EventTypeRepository {
     const profileId = lookupTarget.type === LookupTarget.User ? null : lookupTarget.id;
     const select = {
       ...eventTypeSelect,
-      // TODO:  As required by getByViewHandler - Make it configurable
-      team: {
```

Comment:
Cleanup: Redundant subQuery, teamId already available in eventTypeSelect (does extra query)